README.md

```diff
@@ -25,49 +25,81 @@ This package provides extensible toolbar for Atom.
 * Big *(24px)*
 * Large *(32px)*
 
-# Plugins
+## Plugins
 
 * [toolbar-main](https://atom.io/packages/toolbar-main)
 * [flex-toolbar](https://atom.io/packages/flex-toolbar)
 
-# Integrating toolbar with your package
+## Integrating toolbar with your package
 
-By itself this package just shows empty bar. To add toolbar to your package use this code:
+By itself this package just shows empty toolbar. To add buttons and spacers to the toolbar, use the following code.
 
-```coffeescript
-atom.packages.activatePackage('toolbar')
-  .then (pkg) =>
-    @toolbar = pkg.mainModule
-
-    @toolbar.appendButton 'octoface', 'application:about', 'About Atom'
-    # Adding spacer
-    @toolbar.appendSpacer()
-    # Using custom icon set (Ionicons)
-    @toolbar.appendButton 'gear-a', 'application:show-settings', 'Show Settings', 'ion'
-    # Using custom icon set (FontAwesome)
-    @toolbar.appendButton 'pied-piper-alt', 'application:open-license', 'Open License', 'fa'
-    # Function as a callback
-    @toolbar.appendButton 'alert', ->
-      alert 'foo'
-    , 'Show Alert'
-```
+In `package.json`:
 
-You can also prepend buttons/spacers using `prependButton()` and `prependSpacer()` methods.
+```json
+"package-dependencies": {
+  "toolbar": "^0.1.0"
+},
```

Comment:
Above three lines are actually redundant without https://github.com/travs/atom-package-dependencies which probably shouldn't be used until it's merged into Atom core: https://github.com/atom/apm/issues/288